audio_service/android/src/main/java/com/ryanheise/audioservice/AudioService.java

```diff
@@ -286,32 +281,31 @@ public int onStartCommand(final Intent intent, int flags, int startId) {
     }
 
     public void stop() {
+        notificationCreated = false;
         deactivateMediaSession();
-        stopSelf();
```

Comment:
This is as intended. The app should be in control of triggering each of these methods independently as appropriate for the app. It does this via the `processingState` and `playing` fields.

Reply:
If there is an actual bug here, I would like to see a bug report for it.

Reply:
> This is as intended. The app should be in control of triggering each of these methods independently as appropriate for the app. It does this via the processingState and playing fields.

I see I didn't understand that here. This seems totally reasonable to me now. What I was thinking before is that `stop` should actually stop the service, not "try to stop the service", because if it's foreground it won't be stopped

Reply:
I'll file that as doc issue